doc/source/whatsnew/v0.18.0.txt

```diff
@@ -21,10 +23,67 @@ New features
 ~~~~~~~~~~~~
 
 
+.. _whatsnew_0180.enhancements.moments:
+
+Window functions are now methods
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Window functions have been refactored to be methods on ``Series/DataFrame`` objects, rather than top-level functions, which are now deprecated. This allows these window-type functions, to have a similar API to that of ``.groupby``. See the full documentation :ref:`here <stats.moments>` (:issue:`11603`)
+
+.. ipython:: python
+
+   np.random.seed(1234)
+   df = DataFrame({'A' : range(10), 'B' : np.random.randn(10)})
+   df
 
+Previous Behavior:
 
+.. code-block:: python
 
+   In [8]: pd.rolling_mean(df,window=3)
+   Out[8]:
+       A         B
+   0 NaN       NaN
+   1 NaN       NaN
+   2   1  0.237722
+   3   2 -0.023640
+   4   3  0.133155
+   5   4 -0.048693
+   6   5  0.342054
+   7   6  0.370076
+   8   7  0.079587
+   9   8 -0.954504
+
+New Behavior:
 
+.. ipython:: python
+
+   r = df.rolling(window=3)
+
+These show a descriptive repr, with tab-completion of available methods
+
+.. ipython:: python
+
+   r
+
+The methods operate on this ``Rolling`` object itself
+
+.. ipython:: python
+
+   r.mean()
+
+They provide getitem accessors
+
+.. ipython:: python
+
+   r['A'].mean()
+
+And multiple aggregations
+
+.. ipython:: python
+
+   r.agg({'A' : {'ra' : ['mean','std']},
+          'B' : {'rb' : ['mean','std']}})
```

Comment:
This is not really a useful example I think. It returns:

```
                   A                   B
                  ra                  rb
                mean       std      mean       std
2000-01-01 -1.142297       NaN -0.761535       NaN
2000-01-02 -1.198494  0.079475 -0.730607  0.043739
2000-01-03 -0.746179  0.785446 -0.293707  0.757366
2000-01-04 -0.716306  0.644091 -0.616969  0.894648
...
```

Isn't the purpose of the nested dict not to provide keys for the column names? So if you then provide a list to it, it gives a bit a strange result.

Maybe rather limit to showing the 'different multiple functions to different columns', eg `r.agg({'A': ['mean','std'], 'B': ['median','sum']})`